runtime/bcverify/staticverify.c

```diff
@@ -1537,7 +1554,15 @@ checkMethodStructure (J9PortLibrary * portLib, J9CfrClassFile * classfile, UDATA
 	/* Throw a class format error if we are given a static <init> method (otherwise later we will throw a verify error due to back stack shape) */
 	info = &(classfile->constantPool[method->nameIndex]);
 	if (method->accessFlags & CFR_ACC_STATIC) {
-		if ((CFR_CONSTANT_Utf8 == info->tag) && J9UTF8_DATA_EQUALS("<init>", 6, info->bytes, info->slot1)) {
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
```

Comment:
also need a VT check here

Reply:
Take a look at what I got now. I'm pretty sure that's the VT check you're looking for but just want confirmation.

Reply:
@tajila